### PR TITLE
Add Test for scale to zero using kserve

### PR DESCRIPTION
**Pull request by tarukumar**

### CODE REVIEW

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot [hunk 1]
```diff
@@ -208,6 +207,36 @@ Verify User Can Autoscale Using Concurrency
...
+    ${host}=    Get KServe Inference Host Via CLI    isvc_name=${flan_model_name}   namespace=autoscale-zero
+    ${body}=    Set Variable    '{"text": "At what temperature does liquid Nitrogen boil?"}'
+    ${header}=    Set Variable    'mm-model-id: ${flan_model_name}'
+    Query Model With GRPCURL   host=${host}    port=443
+    ...    endpoint="caikit.runtime.Nlp.NlpService/TextGenerationTaskPredict"
+    ...    json_body=${body}    json_header=${header}
+    ...    insecure=${TRUE}
+    Update Scale To Zero For Knative Serving   serving_name=knative-serving
+    ...    namespace=knative-serving   status=true
+    Scale Number Of Replicas    n_replicas=0    model_name=${flan_model_name}
```
Comment (bdattoma) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot:231]: we probably should rename this kw to "Set Minimum Replicas Number" now

File: ods_ci/tests/Resources/Files/llm/knativeserving_istio.yaml [hunk 2]
```diff
@@ -17,4 +17,7 @@ spec:
       name: autoscaler
   ingress:
     istio:
-      enabled: true
\ No newline at end of file
+      enabled: true
+  config:
+    autoscaler:
+      enable-scale-to-zero: "false"
```
[the remark this answers is not in the document]
Comment (tarukumar) [reply ods_ci/tests/Resources/Files/llm/knativeserving_istio.yaml]: Done

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot (hunk 1 above)
Comment (tarukumar) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot]: Agreed.

**Review by mwaykole — APPROVED** [review]
